consensus/consensus.c

```diff
@@ -107,7 +111,8 @@ retcode_t iota_consensus_init(iota_consensus_t *const consensus,
 
   log_info(CONSENSUS_LOGGER_ID, "Initializing transaction validator\n");
   if ((ret = iota_consensus_transaction_validator_init(
-           &consensus->transaction_validator, SNAPSHOT_TIMESTAMP * 1000,
+           &consensus->transaction_validator,
+           consensus->snapshot.conf.timestamp * 1000,
```

Comment:
could we rename the variable to `timestampSec` so it's clear which resolution it has?
also, `1000uL`